### PR TITLE
feat: Added detailed seat status

### PR DESCRIPTION
**Pull request by cdnninja**
Added detailed seat status.  This is much easier to do here vs in the custom component.  I haven't run a test of this yet, I also don't have seats that report state to test with.

### CODE REVIEW

**Comment by fuatakgun** [comment]
is it working with home assistant without any errors?

**Comment by cdnninja** [comment]
No, I edited the first comment before you posted but noticed in my mobile phone it doesn't show the edit.  Odd. 

I am going to do a round of testing with this plus #21 at the same time.  I also can't test the values as my car doesn't show seat status.

**Comment by cdnninja** [comment]
I have now tested this with home assistant core.  Reader for merge if you are good with it.  As mentioned I can't see the sensors or data though.  It doesn't error for those without.

**Comment by cdnninja** [comment]
As a note #21 will fix the failing python builds.

File: hyundai_kia_connect_api/KiaUvoApiEU.py [hunk 1]
```diff
@@ -216,18 +217,18 @@ def update_vehicle_with_cached_state(self, token: Token, vehicle: Vehicle) -> No
         vehicle.side_mirror_heater_is_on = get_child_value(
             state, "vehicleStatus.sideMirrorHeat"
         )
-        vehicle.front_left_seat_heater_is_on = get_child_value(
+        vehicle.front_left_seat_status = SEAT_STATUS[get_child_value(
```
Comment (fuatakgun) [inline hyundai_kia_connect_api/KiaUvoApiEU.py:220]: overall, should we stick with `front_left_seat_heater_status` and `SEAT_HEAT_STATUS`? or do we store any state other than heat here?

**Review by fuatakgun — CHANGES_REQUESTED** [review]

File: hyundai_kia_connect_api/KiaUvoApiEU.py (hunk 1 above)
Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:220]: We store states including cool.   The constant file has the text states for your reference.

Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiEU.py:220]: https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/783b6abace0fc9154f9ea4c93f4aabb43dbaf440/hyundai_kia_connect_api/const.py#L22

**Comment by cdnninja** [comment]
Ready for review and merge.

**Pull request merged**